<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWidget</class>
 <widget class="QWidget" name="UserWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>10</width>
    <height>10</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Smart Home Control Center</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QWidget" name="TopStateBar" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="logoLabel">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/new/prefix1/resources/logo.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="networkLabel">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../resources.qrc">:/new/prefix1/resources/network_online.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="mesgButton">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>32</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>10</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>消息</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/prefix1/resources/message_icon.png</normaloff>:/new/prefix1/resources/message_icon.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="userButton">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string>用户</string>
         </property>
         <property name="icon">
          <iconset resource="../resources.qrc">
           <normaloff>:/new/prefix1/resources/user_avatar.png</normaloff>:/new/prefix1/resources/user_avatar.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timeLabel">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>设备控制</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>环境监测</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>场景模式</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>安全监控</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>系统设置</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
